package.json

```diff
@@ -1,125 +1,221 @@
-          "description": "Sourcery token. You can find your token at https://sourcery.ai/dashboard"
-        },
-        "sourcery.ruleType.refactorings": {
-          "type": "boolean",
-          "default": true,
-          "description": "Show refactorings."
-        },
-        "sourcery.ruleType.suggestions": {
-          "type": "boolean",
-          "default": true,
-          "description": "Show suggestions."
-        },
-        "sourcery.ruleType.comments": {
-          "type": "boolean",
-          "default": true,
-          "description": "Show comments."
-        }
-      }
-    }
-  },
-  "main": "./out/extension",
-  "scripts": {
-    "vscode:prepublish": "tsc -p ./",
-    "vscode:uninstall": "node ./out/uninstall",
-    "compile": "tsc -watch -p ./",
-    "vscode": "npm run vscode:prepublish && VSCODE=$(which code-insiders || which code || echo echo ERROR: neither the code nor code-insiders vscode executable is installed); USER=dummy-dont-share-vscode-instance \"$VSCODE\" --user-data-dir=$PWD/.vscode-dev/user-data --verbose --extensionHomePath=$PWD/.vscode-dev/extensions --extensionDevelopmentPath=$PWD $*",
-    "vscode_local": "SOURCERY_EXECUTABLE=../core/run-sourcery.sh yarn run vscode"
-  },
-  "devDependencies": {
-    "@types/mocha": "^5.2.7",
-    "@types/node": "^12.11.7",
-    "@types/vscode": "^1.30.0",
-    "mocha": "^7.0.0",
-    "ovsx": "^0.2.0",
-    "tslint": "^6.1.0",
-    "typescript": "^3.8.3",
-    "version-bump-prompt": "^6.1.0",
-    "vsce": "^1.95.1",
-    "vscode-test": "^1.3.0"
-  },
-  "dependencies": {
-    "vscode-languageclient": "^5.2.1"
-  }
-}
+	"name": "sourcery",
+	"displayName": "Sourcery",
+	"description": "Refactor Python instantly with Sourcery",
+	"author": "Sourcery AI",
+	"repository": "https://github.com/sourcery-ai/sourcery-vscode",
+	"bugs": {
+		"url": "https://github.com/sourcery-ai/public/issues"
+	},
+	"license": "MIT",
+	"version": "0.12.1-dev.1",
+	"publisher": "sourcery",
+	"icon": "sourcery-icon.png",
+	"keywords": [
+		"Python",
+		"refactoring"
+	],
+	"engines": {
+		"vscode": "^1.65.0"
+	},
+	"categories": [
+		"Other"
+	],
+	"activationEvents": [
+		"onLanguage:python",
+		"onStartupFinished"
+	],
+	"contributes": {
+		"walkthroughs": [
+			{
+				"id": "sourceryWelcome",
+				"title": "Get started with Sourcery",
+				"description": "Your first steps to try out the powerful tools and features that the Sourcery extension has to offer!",
+				"when": "workspacePlatform != webworker",
+				"steps": [
+					{
+						"id": "sourcery.openWelcomeFile",
+						"title": "Get your first recommendation",
+						"description": "Get started by opening our tutorial \n[Open tutorial](command:toSide:sourcery.welcome.open)\n Put the cursor over the underlined code to see the refactoring suggestion. \n You can see all Sourcery's recommendations in the [Problems Pane](command:workbench.actions.view.problems)",
+						"media": {
+							"markdown": "walkthrough/view_welcome_file.md"
```

Comment:
Please add something to this - you could probably move some of the steps out of custom rules, and replace the screenshots

Reply:
Will address this in follow-up PR